### PR TITLE
Main repository readme

### PR DESCRIPTION
**Pull request by Irma12**
This is just a draft for our readme for the main repo, so I will keep on adding content after you suggest changes/additions

### CODE REVIEW

**Review by Xantier — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+So...feel ready to jump in? Let's do this. 💯 👏
+
+Start by reading repository README to get set up for local development. If you need help, just jump into [our Discord chatroom](https://discord.gg/3S4xrW7B).
+
+## Coding Guidelines
+
+We use the backstage-cli to build, serve, lint, test and package all the plugins.
+
+As such, the [same coding guidelines mostly apply](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#coding-guidelines).
```
Comment (Xantier) [inline CONTRIBUTING.md:52]:
```suggestion
As such, the [same coding guidelines as in Backstage repository mostly apply](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#coding-guidelines).
```

**Review by iain-b — APPROVED** [review]
looks good to me

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+## Code of Conduct
+
+We subscribe to the [Spotify FOSS code of conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md) which is used by the Backstage project.
+
+If you experience or witness unacceptable behavior—or have any other concerns—please report it by contacting us via [foss@roadie.io](mailto:foss@roadie.io).
+
+## Security Issues?
+
+See [SECURITY.md](https://github.com/RoadieHQ/backstage-plugin-github-pull-requests/blob/main/SECURITY.md)
```
Comment (dtuite) [inline CONTRIBUTING.md:66]: This link will need updating. Can be in a separate PR.

File: README.md [hunk 3]
```diff
@@ -1,10 +1,99 @@
-# [Backstage](https://backstage.io)
+This monorepo will contain all plugins we, at Roadie, have worked on up until now. Currently we are in the process of integration so for now you may not find all the plugins in this repository, so you can search for the ones we still haven't integrated [here](https://github.com/RoadieHQ?q=backstage-plugins&type=public&language=&sort=stargazers) or look them up on [npmjs](https://npmjs.com).
 
-This is your newly scaffolded Backstage App, Good Luck!
 
-To start the app, run:
+## Getting Started
```
Comment (dtuite) [inline README.md:4]: This is great. Well done ❤️

**Review by dtuite — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (Irma12) [reply CONTRIBUTING.md]: Fixed :)
